core/runner.py

```diff
@@ -83,16 +82,20 @@ def _train(self, job):
         starting from the initial model state specified, until a stopping
         condition is met, and using the hyper-parameters specified.
 
-        Returns the updated model weights, the weighting factor omega, and stats
-        about the training job.
+        Returns a DMLResult with the updated model weights, the weighting factor 
+        omega, and stats about the training job.
 
         NOTE: Uses the same hyperparameters and labeler for training and
         validating during 'avg_type' of type 'val_acc'.
 
         NOTE2: Assumes 'initial_weights' are the actual weights and not a path.
         """
         # Get the right dataset iterator based on the averaging type.
-        serialized_model, framework_type, initial_weights, hyperparams, labeler = job.serialized_model, job.framework_type, job.weights, job.hyperparams, job.label_column_name
+        serialized_model = job.serialized_model
+        framework_type = job.framework_type
+        initial_weights = job.weights
+        hyperparams = job.hyperparams
+        labeler = job.label_column_name        
```

Comment:
Nitpick but why initialize all these variables? I think it's more readable and less lines of code to just use `job.*`